lib/site_encrypt/phoenix.ex

```diff
@@ -18,6 +30,7 @@ defmodule SiteEncrypt.Phoenix do
     )
   end
 
+  @spec restart_endpoint(config) :: tuple()
```

Comment:
This one is a bit tricky. I think the best course of actions would be to make the function always return `:ok` and then specify that as the return type.